### PR TITLE
Shows and Hides the Remove Link for Creator Based on Number of Creators

### PR DESCRIPTION
**Pull request by mtribone**
Adds javascript to show or hide the remove button based on the number of creators. Passes variable with inline style to show/hide to the creator partial.

### CODE REVIEW

**Comment by mtribone** [comment]
One creator no remove link

screen shot 2017-11-16 at 3 48 39 pm

**Comment by mtribone** [comment]
Two creators with remove links

screen shot 2017-11-16 at 3 50 24 pm

**Comment by mtribone** [comment]
Existing work with two creators and remove links

screen shot 2017-11-16 at 3 52 08 pm

**Comment by mtribone** [comment]
Removing first creator on an existing work with 2 creators and no remove link


screen shot 2017-11-16 at 3 53 14 pm

File: app/views/records/edit_fields/_creator.html.erb [hunk 1]
```diff
@@ -9,14 +9,20 @@
            placeholder="Search for an existing creator" />
   </div>
 
+  <% if @form.creators.count == 1
+       remove_button_style = 'display: none'
+     else
+
```
Comment (carolyncole) [inline app/views/records/edit_fields/_creator.html.erb]: @mtribone If we are going to be explicit this should be remove_button_style = ''

File: app/views/records/edit_fields/_creator.html.erb [hunk 2]
```diff
@@ -9,14 +9,20 @@
...
   <% @form.creators.each_with_index do |creator_form, index| %>
     <% id_base = "#{@form.model_class_name}[creators][#{index}]"%>
 
     <div class="well creator_inputs">
-      <button type="button" class="btn btn-link remove remove-creator">
-        <span class="glyphicon glyphicon-remove"></span>
-        <span class="controls-remove-text">Remove</span>
-      </button>
+        <button type="button" class="btn btn-link remove remove-creator" style="<%= remove_button_style %>" >
+          <span class="glyphicon glyphicon-remove"></span><span class="controls-remove-text">Remove</span>
```
Comment (carolyncole) [inline app/views/records/edit_fields/_creator.html.erb]: @mtribone was there a reason to change the spacing here?  It muddies the PR.

File: app/views/records/edit_fields/_creator_template.html.erb [hunk 3]
```diff
@@ -3,7 +3,7 @@
     <button type="button" class="btn btn-link remove remove-creator">
       <span class="glyphicon glyphicon-remove"></span><span class="controls-remove-text">Remove</span>
     </button>
-    <input type="hidden" name="<%= @form.model_class_name %>[creators][{{index}}][id]" id="<%= @form.model_class_name %>{{index}}_id" value="{{id}}">
+      <input type="hidden" name="<%= @form.model_class_name %>[creators][{{index}}][id]" id="<%= @form.model_class_name %>{{index}}_id" value="{{id}}">
```
Comment (carolyncole) [inline app/views/records/edit_fields/_creator_template.html.erb]: @mtribone any reason to change this line.  The only thing I can see is spacing and it is not aligned with the code above

**Review by carolyncole — CHANGES_REQUESTED** [review]

File: app/views/records/edit_fields/_creator.html.erb (hunk 1 above)
Comment (mtribone) [reply app/views/records/edit_fields/_creator.html.erb]: @cam156 ah. copy/pasta. double quotes

**Review by carolyncole — APPROVED** [review]
waiting on travis

**Pull request merged**